### ISSUE TITLE
Stuck at "Idena Node is starting.."

### ISSUE THREAD

**Issue by ligi** [idena-network/idena-desktop#737]
when opening idena currently I just see:

![image](https://user-images.githubusercontent.com/111600/142756613-520b79ea-8c02-4fca-8b58-c9fe2192ba8b.png)

in the console I see:

```
✓ ~ idena-desktop                                                                                            Checking for update
Error: Error: ENOENT: no such file or directory, open '/opt/Idena/resources/app-update.yml'
```

I was running a VPS for a long time - but for a test wanted to switch to the build-in node - this caused the problem I think. It should recover from this Error I think.

**Comment by aidenaio** [comment]
yes this is a problem with the built-in node, moving to the node project
